public/javascripts/Admin/src/Admin.js

```diff
@@ -434,8 +434,18 @@ function Admin(_, $) {
                 vega.embed("#severity-histograms", chart, opt, function(error, results) {});
                 vega.embed("#severity-histograms2", chart2, opt, function(error, results) {});
             });
-
             $.getJSON('/adminapi/neighborhoodCompletionRate', function (data) {
+                // Determine height of the chart based on size of data
```

Comment:
A reminder to end comments with a period, as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide) :upside_down_face:

Reply:
And instead of "size of data", it might be more clear to say "the number of neighborhoods"

Reply:
Sorry about that! I'll go back and fix this.

Reply:
I'll also take a look into the x-axis scrolling issue and get those changes pushed asap!